amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -30,6 +30,7 @@ THE SOFTWARE.
 #endif
 
 #include "internal_rpp.h"
+#define NUM_OF_DIMS 5
```

Comment:
@fiona-gladwin  can you rename this to rpp_max_tensor_dims or similar? We already have another tensor dimensions set to max 6 [here](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/amd_openvx/openvx/ago/ago_internal.h#L107). Would want to avoid any confusion

Reply:
Yes.